pkg/network/go/bininspect/pclntab_test.go

```diff
@@ -0,0 +1,62 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2024-present Datadog, Inc.
+
+//go:build linux
```

Comment:
build tag 'test' is missing

Reply:
no need, the name suffix is `_test.go` which guarantees the code never reaches to the compiled binaries.

Similarly, if the file has a suffix of `_linux`, `windows`, etc. it is equivalents to `//go:build linux` or `//go:build windows`

Reply:
> To write a new test suite, create a file that contains the TestXxx functions as described here, and give that file a name ending in "_test.go". The file will be excluded from regular package builds but will be included when the "go test" command is run.

taken from the first paragraph of https://pkg.go.dev/testing